### PR TITLE
Tabs: Add orientation option

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js [hunk 1]
```diff
@@ -244,6 +246,38 @@ QUnit.module('General', () => {
...
+    QUnit.test('the tabs element must have a correct orientation class in runtime change', function(assert) {
+        const $element = $('#tabs').dxTabs({
+            items: [1, 2, 3],
+        });
+        const tabs = $element.dxTabs('instance');
+
+        assert.ok($element.hasClass(TABS_HORIZONTAL_CLASS));
+
+        tabs.option('orientation', 'vertical');
+        assert.ok($element.hasClass(TABS_VERTICAL_CLASS));
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js:276]: There is no assert that other class was removed from element

File: packages/devextreme/js/ui/tabs.js [hunk 2]
```diff
@@ -123,14 +131,23 @@ const Tabs = CollectionWidget.inherit({
...
+        this.$element().addClass(elementClasses);
+    },
+
     _init: function() {
         this.callBase();
-
         this.setAria('role', 'tablist');
-
-        this.$element().addClass(TABS_CLASS);
+        this._setBaseElementClasses();
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/ui/tabs.js]: I'd rather to add dx-tabs class here
```
this.$element().addClass(TABS_CLASS)
```
The logic with toggling orientation classes to extract to separate method as follows
``` 
toggleOrientationClass(orientation) { ...}
```
and use it inside optionChanged method and other places where it needs

**Review by EugeniyKiyashko — DISMISSED** [review]

File: packages/devextreme/js/ui/tabs.js (hunk 2 above)
Comment (marker-dao) [reply packages/devextreme/js/ui/tabs.js]: Nice point, it really looks better) Thanks!

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js:276]: Fixed [[here](116ba1d0d6e538ade3847fc0196b07a792bc5990)].

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by AlexanderMoiseev — APPROVED** [review]

**Review by vladaskorohodova — APPROVED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**